packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -186,6 +186,15 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
         emit Unpaused(msg.sender);
     }
 
+    /**
+     * @notice Computes the minimum gas limit for a deposit. The minimum gas limit
+     *         linearly increases based on the size of the calldata. This is to prevent
+     *         users from creating L2 resource usage without paying for it.
+     */
+    function minimumGasLimit(uint64 _byteCount) public pure returns (uint64) {
+        return _byteCount * 16 + 21000;
+    }
```

Comment:
nit:

```suggestion
    function minimumGasLimit(uint256 _byteCount) public pure returns (uint64) {
        return uint64(_byteCount) * 16 + 21000;
    }
```

I can take it or leave it, but it feels like we can probably make integration a bit easier to handling the casting here ourselves.

Reply:
The cast can cause overflow in this case which is why I didn't go with this pattern

Reply:
Ahh, good call.